pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/transformer/MetsData.scala

```diff
@@ -108,17 +117,30 @@ case class MetsData(
 
   private val thumbnailDim = "200"
 
-  private def thumbnail(maybeLicense: Option[License], bnumber: String) =
+  private def thumbnail(maybeLicense: Option[License],
+                        bnumber: String,
+                        maybeFileId: Option[String]) =
     for {
       location <- thumbnailLocation
       url <- buildThumbnailUrl(location, bnumber)
     } yield
       DigitalLocation(
         url = url,
         locationType = LocationType("thumbnail-image"),
+        imageSourceIds = maybeFileId
```

Comment:
Not completely convinced whether this needs to be here on `thumbnail` but decided it was more consistent

Reply:
I'm starting to think that we might need to create _special_ digital locations that don't get displayed in the API so that we are not coupling to what gets extracted as a thumbnail or as an item for api display purposes. 
For example, for MIRO images we extract the `thumbnail` as a `DigitalLocation` with the url of a 300x300 image and we also extract the `DigitalLocation` of the `iiif-image` api of the image, and I think this is what we want. 
But for METS we don't currently extract a `iiif-image` url even though we definitely can. So I'm starting to think that we should just extract a field `image` with a `sourceIdentifier` and a `DigitalLocation` with a `url` pointing to a `iiif-image` api. This way we could make the merger always look at the `image` field

Reply:
If we extract just one `image` then how do we deal with multiple images on a source work (eg for METS)? Would that be as simple as making it an `images` field with a list?

I had been thinking that these `imageSourceIds` might get discarded when displayed in the API but agreed that a special location for them is probably better.

Reply:
Yep, sorry a field like `images: List[Image]]` where `Image(sourceIdentifier:SourceIdentifier, location:DigitalLocation)`